src/test/java/com/contentgrid/gateway/GatewayApplicationTest.java

```diff
@@ -24,6 +30,23 @@ void contextLoads() {
             "spring.main.cloud-platform=kubernetes"
     })
     class RuntimeWithKubernetes {
+
+        @TestConfiguration
+        static class TestConfig {
+
+            @Bean
+            @Primary
+            KubernetesResourceWatcherBinding dummyKubernetesResourceWatcherBinding() {
+                return Mockito.mock(KubernetesResourceWatcherBinding.class);
+            }
+
+            @Bean
+            @Primary
+            KubernetesServiceDiscovery dummyKubernetesServiceDiscovery() {
+                return Mockito.mock(KubernetesServiceDiscovery.class);
+            }
+        }
+
```

Comment:
The value of the test becomes a little questionable.

Reply:
Why do you think so ? What do you think is the purpose of this test ?